### PR TITLE
if-statemnent unit conversion. read_geoms generalize for file type

### PR DESCRIPTION
**Pull request by Santonia27**

### CODE REVIEW

File: hydromt_fiat/fiat.py [hunk 1]
```diff
@@ -1286,19 +1286,24 @@ def read_geoms(self):
 
         fns = glob.glob(Path(self.root, "geoms", "*.geojson").as_posix())
         if self.spatial_joins["aggregation_areas"]:
-            fns_aggregation = glob.glob(Path(self.root, "geoms", "aggregation_areas","*.geojson").as_posix())
-            fns.append(fns_aggregation[0])
+            fns_aggregation = glob.glob(Path(self.root, "geoms", "aggregation_areas/*").as_posix())
```
Comment (panosatha) [inline hydromt_fiat/fiat.py:1289]: will this now work if this is a shapefile wiht multiple files?

Comment (Santonia27) [reply hydromt_fiat/fiat.py:1289]: a shapefile with multiple layers or multiple files in the folder? 
I tested it with 2 files in the aggregation areas and that seemed to work

File: hydromt_fiat/fiat.py [hunk 2]
```diff
@@ -1284,23 +1284,40 @@ def read_geoms(self):
             ]
             self.exposure.read_geoms(exposure_fn)
 
+        
         fns = glob.glob(Path(self.root, "geoms", "*.geojson").as_posix())
         if self.spatial_joins["aggregation_areas"]:
-            fns_aggregation = glob.glob(Path(self.root, "geoms", "aggregation_areas","*.geojson").as_posix())
-            fns.append(fns_aggregation[0])
+            fns_aggregation = glob.glob(Path(self.root, "geoms", "aggregation_areas/*").as_posix())
```
Comment (panosatha) [inline hydromt_fiat/fiat.py:1290]: I dont understand why you are reading the location of the geometries from this pre-defined path, while these should be saved in the spatial_joins obejct. You could just read the paths from there right?

**Review by panosatha — CHANGES_REQUESTED** [review]
The shapefile seems to work fine, but I noticed something in the way that you read in the paths of the geometries that puzzled me. See commnet above

**Review by panosatha — APPROVED** [review]
LGTM

**Pull request merged**